include/host/hash_util.hpp

```diff
@@ -18,5 +18,7 @@ void
 hash_extend_page(hash_ctx_t* hash_ctx, const void* ptr);
 void
 hash_finalize(void* md, hash_ctx_t* hash_ctx);
+/*
```

Comment:
Please remove this if `printHash` is not used.

Reply:
Done. Someone commented out the definition in hash_util.cpp so I just commented out the declaration in hash_util.hpp. Now both the definition and the declaration are removed in this PR.

ee52d1bd2a923eda33e92fd3e669b805e90b1d8f